README.md

```diff
@@ -1,2 +1,27 @@
-# support
-For questions on how to get IPFS up and running smoothly
+# Support
+
+For questions on how to get IPFS up and running smoothly.
```

Comment:
I find that github issues are difficult to search through, I know we've said no before, but a wiki might be nice